snc.sh

```diff
@@ -233,6 +233,10 @@ function renew_certificates() {
     cli_image=$(${OC} adm release -a ${OPENSHIFT_PULL_SECRET_PATH} info ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE} --image-for=cli)
     ${YQ} write kubelet-bootstrap-cred-manager-ds.yaml.in spec.template.spec.containers[0].image ${cli_image} >kubelet-bootstrap-cred-manager-ds.yaml
 
+    # Enable the network time sync and set the clock back to present on host
+    sudo date -s '1 day'
+    sudo timedatectl set-ntp on
+
```

Comment:
Guillaume was not keen on changing the time on some of his test machines, I thought the agreement was to make cert renewal optional (for test purposes)?

Reply:
@cfergeau should I introduce a env variable which is set to true by default and if user don't want the cert rotation then set it to false?

Reply:
yes, sounds good to me, @guillaumerose, would that fit your usecase?

Reply:
yes, it would be awesome! I use my SNC build machine as my regular machine :) It will save my https connections!